### ISSUE TITLE
How to use Social login?

### ISSUE THREAD

**Issue by chewtoys** [danielfsousa/express-rest-es2017-boilerplate#24]
I assume you have to do it on client side?

Simple JS example would be nice...

**Comment by danielfsousa** [comment]
That's right, you need to send the "access_token" to the api. Both facebook and google docs have good examples that you can take a look:

https://developers.google.com/identity/sign-in/web
https://developers.facebook.com/docs/facebook-login/web

**Comment by nengine** [comment]
Hello,  If I want to test REST api from Postman with just jwt please let me know how should I do? Thanks.
